src/components/contact/Contact.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { translate, Interpolate } from 'react-i18next';
+
+import '../../i18n/i18n';
+import styles from './assets/contact.scss';
+import peopleSearch from './assets/people-search.svg';
+import SocialMedia from './SocialMedia';
+
+const Contact = ({ t }) => {
+  const EmailLink = <a target="_blank" href={`mailto:${process.env.REACT_APP_EMAIL}`} className={styles.email}>{process.env.REACT_APP_EMAIL}</a>;
```

Comment:
Usually you use Pascal Case for React components. Technically though, this is not a React component, but instead a reference to an element. Meaning, you can't instantiate it as `<EmailLink />`. Instead, you would wrap it in curly braces, like `{EmailLink}`, in a React component. I think it would be fine to leave it as camel case.